doc/source/publications.rst

```diff
@@ -0,0 +1,29 @@
+############
+Publications
+############
+
+This section regroups the peer-reviewed journal articles that **describe** core functionnalities of Lethe or that **use** Lethe. The publications are grouped by the year of the publications and displayed in the Chicago format.
+
+
+
+2022
+----
+
+* `Barbeau, Lucka, Stéphane Étienne, Cédric Béguin, and Bruno Blais. "Development of a high-order continuous Galerkin sharp-interface immersed boundary method and its application to incompressible flow problems." Computers & Fluids 239 (2022): 105415. <https://doi.org/10.1016/j.compfluid.2022.105415>`_
```

Comment:
I think we should write all the authors in the same way and I would remove the comma or the "and" before the last author:

```suggestion
* `Lucka Barbeau, Stéphane Étienne, Cédric Béguin and Bruno Blais. "Development of a high-order continuous Galerkin sharp-interface immersed boundary method and its application to incompressible flow problems." Computers & Fluids 239 (2022): 105415. <https://doi.org/10.1016/j.compfluid.2022.105415>`_
```

Reply:
Yeah this first name inversion is a particularity of the Chicago standard for citations. Do you have another standard your would suggest we use instead?

Reply:
I personally prefer the IEEE style, but if you prefer the Chicago citation style, that is also fine :)

Reply:
I am cool with IEEE style, is there a way to automatically generate the IEEE style reference from the bibtex file? I used the Chicago style because it's my default in google scholar (and I am a lazy bastard)

Reply:
Seems like an online converted exists :)! https://www.bibtex.com/c/bibtex-to-ieee-converter/